File: jour_15/jour15.pas
program sum_calcul;
var 
    i,somme,number : integer;
begin
    write('Entrez la valeur : ');
    readln(number);
    somme := 0;
    for i:= 0 to number do
        begin 
            somme := somme + i;
        end;
    write('La somme des entiers inferieurs ou égales a ',number,' est : ',somme);
end.

File: jour_15/jour15.c
#include<stdio.h> 
int main(){
    int number;
    printf("Entrez une valeur : ");
    scanf("%d",&number);
    printf("La somme des entiers inférieurs ou égale a %d est : %d\n",number,(number*(number + 1))/2);
    return 0;
}